plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -361,7 +363,32 @@ func ClusterRoles() []rbac.ClusterRole {
 				eventsRule(),
 			},
 		},
+		{
+			// a role making the csrapprover controller approve a node client CSR
+			ObjectMeta: metav1.ObjectMeta{Name: "system:csr-approver:node-client-csr"},
```

Comment:
I suggest just using the name of the sar verbs. The final csr is redundent. E.g.:
```
system:csr-approver:nodeclient
system:csr-approver:selfnodeclient
system:csr-approver:selfnodeserver
```